src/main/java/com/dev/cinema/controllers/ShoppingCartController.java

```diff
@@ -37,15 +37,15 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     public void addMovieSessionToShoppingCart(
```

Comment:
Bad URL mapping (@PostMapping("/add-movie-session/{movieSessionId}")). Better - just a @PostMapping. I don't understand why we need to pass movieSessionId in post mehod

Reply:
done